src/Operation/BulkWrite.php

```diff
@@ -348,6 +357,10 @@ private function createBulkWriteOptions()
             $options['bypassDocumentValidation'] = $this->options['bypassDocumentValidation'];
         }
 
+        if (isset($this->options['let'])) {
+            $options['let'] = (object) $this->options['let'];
```

Comment:
Thank you for remembering to use an `(object)` cast here. That's always helpful for options that we know are documents, even if not explicitly required by PHPC (since it will be serialized to a BSON document regardless).